### ISSUE TITLE
Merge invitation list and stats table

### ISSUE THREAD

**Issue by Jackseed** [blockframes/blockframes#4257]
## Description 
Invitation list doesn't display enough information for the sales agent and the updated stats table will have almost everything needed. Best is to merge these 2 lists.
![Invitations.png](https://images.zenhubusercontent.com/5c6a89ac8de1b31d5443ab99/aa384645-4605-4451-af7a-82aeb23396e2)
![Attendees.png](https://images.zenhubusercontent.com/5c6a89ac8de1b31d5443ab99/84716dd8-84ef-453b-9b32-3eafaec70000)

## Specs
The new invitation table should then be visible as soon as there is one invitation sent. It should include the following columns, in that order:
- userFirstName + userLastName  | label: Name
- userEmailAddress | label: Email Address
- orgName | label: Company Name
- orgActivity | label: Company Activity
- orgCountry | label: Country
- invitationType (either "Buyer request" or "Sales agent invitation") | label: Type
- invitation status  (pending, accepted, declined) | label: Status
- attendeed (yes / no) | label: Attended  (comes from analytics)
- action button

Each line of the table will change a lot through time depending on user status as they could have no name, org, or can't attend before the event takes place. If there is no info, a "-" should be displayed. 

The action button should open a list menu. If that's possible, the actions should differ depending on the invitation type
_Buyer request_
- Accept
- Decline

_Sales agent invitation_
- Cancel 

If that's not possible, please tell me so we open a different issue to find a solution. 

The "Number of attendees" card should be displayed below the "Invitations" title, before the invitation input and be visible only after the event when we receive Analytics. 

The guest amount should also stay below the invitation input and we shall rename "# yes" / "# no" / "# unanswered" to "# accepted" "# declined" / "# pending". 

## Acceptance criteria
- [ ] Remove invitation list
- [ ] Replace it with stats table
- [ ] Remove the stats section
- [ ] Add new table columns
- [ ] Action list
- [ ] Move attendees card
- [ ] Change guest amounts labels

**Comment by Jackseed** [comment]
@2amarkina I think this is no longer needed with the new tab structure, could you confirm so we close it?

**Comment by 2amarkina** [comment]
yes!
